documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -6,6 +6,7 @@
 
 * The `detect.diagnostic.extended` property, that was deprecated in [solution_name] 8.x, has been removed. `detect.diagnostic` can be used instead.
 * The Ephemeral Scan Mode, that was deprecated in [solution_name] 8.x, has been removed in favor of Stateless Scan Mode. See the [Stateless Scans page](runningdetect/statelessscan.md) for further details.
+* The -de command line option has been removed in favor of -d as the two options have been identical since [solution_name] 8.x. 
```

Comment:
Should we say what the function of -de/-d options are here in the release note? Or maybe we could hyperlink the "-d" to where that very option is mentioned in the docs (similar to what we have on line 7). A user affected by this change would/should be familiar anyhow but just a thought.